tests/Aspire.Hosting.Tests/AddParameterTests.cs

```diff
@@ -13,7 +15,7 @@ public class AddParameterTests
     public void ParametersAreHiddenByDefault()
     {
         var appBuilder = DistributedApplication.CreateBuilder();
-        appBuilder.Configuration["Parameters:pass"] = "pass1";
+        appBuilder.Configuration[$"Parameters:pass"] = "pass1";
```

Comment:
I'm not sure I understand why these changes were made. This doesn't do anything, right?

Reply:
That's a shadow of a previous commit that was undone. I'll fix.